ui/patches/react-archer+4.1.0.patch

```diff
@@ -5,10 +5,10 @@ index 4f6480f..3fc574a 100644
 \ No newline at end of file
-+const CURVE_RADIUS = '10';
++const CURVE_RADIUS = 10;
 +// This draws an l line segment with a curved corner of radius `CURVE_RADIUS`
 +function customLineSegment(e) {
-+  return `M${e.xStart},${e.yStart} ${e.xStart},${e.yEnd - CURVE_RADIUS} A${CURVE_RADIUS},${CURVE_RADIUS} 0 0,0 ${e.xStart + 10},${e.yEnd} M${e.xStart + CURVE_RADIUS},${e.yEnd} ${e.xEnd},${e.yEnd}`;
++  return `M${e.xStart},${e.yStart} ${e.xStart},${e.yEnd - CURVE_RADIUS} A${CURVE_RADIUS},${CURVE_RADIUS} 0 0,0 ${e.xStart + CURVE_RADIUS},${e.yEnd} M${e.xStart + CURVE_RADIUS},${e.yEnd} ${e.xEnd},${e.yEnd}`;
 +}
 +
```

Comment:
Wow, nice job tracking this down. Do you think we can get this fix upstreamed to react-archer?

Reply:
Ah, I got very confused and thought the patch was introduced in this PR.

@jgzuke pretend the original comment was addressed at you 🙂